examples/multistage/Singularity

```diff
@@ -0,0 +1,43 @@
+# Build binary using base image with golang installed
+Bootstrap: docker
+From: golang:1.12.3-alpine3.9
+Stage: one
+
+
+%post
+        # prep environment
+        export PATH="/go/bin:/usr/local/go/bin:$PATH"
+        export HOME="/root"
+        cd /root
+
+        # insert source code, could also be copied from host with %files
+        cat << EOF > hello.go
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Printf("Hello World!\n")
```

Comment:
```suggestion
    fmt.Printf("Hello World From Stage One!\n")
```

Reply:
I think its best to leave this as is since its already clear the application was build in stage one